templates/new/config/config.exs

```diff
@@ -13,8 +13,33 @@ config :nerves, :firmware, rootfs_overlay: "rootfs_overlay"
 # docs for separating out critical OTP applications such as those
 # involved with firmware updates.
 config :shoehorn,
-  init: [:nerves_runtime],
-  app: Mix.Project.config()[:app]
+  init: [:nerves_runtime<%= if init_gadget? do %>, :nerves_init_gadget<% end %>],
+  app: Mix.Project.config()[:app]<%= if init_gadget? do %>
+
+# Uncomment to authorize the device to receive firmware using your public key.
+# See https://hexdocs.pm/nerves_firmware_ssh/readme.html for more information
+# on configuring nerves_firmware_ssh.
+ 
+# config :nerves_firmware_ssh,
+#   authorized_keys: [
+#     File.read!(Path.join(System.user_home!, ".ssh/id_rsa.pub"))
+#   ]
+
+# Uncomment to use Ringlogger as the logger backend and remove :console.
+# See https://hexdocs.pm/ring_logger/readme.html for more information on
+# configuring ring_logger.
+
+# config :logger, backends: [RingLogger]
+
+# Uncomment to alter the configuration for nerves_init_gadget.
+# See https://hexdocs.pm/nerves_init_gadget/readme.html for more information.
+
+# config :nerves_init_gadget,
+#   ifname: "usb0",
+#   address_method: :linklocal,
+#   mdns_domain: "nerves.local",
+#   node_name: nil,
+#   node_host: :mdns_domain<% end %>
```

Comment:
I almost always add `ssh_console_port: 22` now to get ssh access to iex. `nerves_init_gadget` should probably make this the default, but until then...

Reply:
Should we leave this section commented out or should it also be uncommented? I like the idea of having them uncommented, even though they are currently the default. What do you think?

Reply:
I like them uncommented even if they're the default. I always forget the key names.